### ISSUE TITLE
make tests trigger for external contributors

### ISSUE THREAD

**Issue by motatoes** [diggerhq/digger#1293]
In most of our tests we listen for:

```
  pull_request:
    types: [opened, reopened]
```

It seems to not trigger for external cotributors. We should replace it with:

```
  pull_request:
```

because that seems to work better

**Comment by JoshuaJackson-jobvite** [comment]
Will likely want to have something like:

```
    if: github.event.pull_request.head.repo.full_name == github.repository
```
in the job definition header as well to avoid forks running tasks in the source repo etc..that could expose things that might not want to be exposed.

**Comment by motatoes** [comment]
Yeah so true, that will be needed. I think the only one that exposes secrets is the e2e tests, rest don't contain sensitive data :)
